### ISSUE TITLE
Linter complain on try/catch syntax

### ISSUE THREAD

**Issue by mfornet** [duaraghav8/Ethlint#293]
**Description**
It should pass but complains about syntax error during linting:

```
contracts/test/Tester.sol
  34:13    error    Syntax error: unexpected token t
```

For the code:

```solidity
    function tryWithdrawAndAvoidFail(bool toNear) external {
        try this.withdrawAndFail(toNear) {
            require(false);
        } catch {
        }
    }
```

Line 34, is the one containing the `try` statement

**Operating System**
MacOS 10.15.7

**Linter version**
Solium version 1.2.5

**Comments**
If you'd like to share any additional information that doesn't fit in above sections or suggest a solution to the maintainers.

**Priority**
`Medium`
